addons/implied-repos/common/src/main/java/org/commonjava/indy/implrepo/change/ImpliedRepositoryDetector.java

```diff
@@ -459,6 +459,42 @@ else if ( !config.isIncludeSnapshotRepos() && !repo.isReleasesEnabled() )
                 else
                 {
                     logger.debug( "Found existing RemoteRepositories: {}", rrs );
+
+                    for ( final RemoteRepository rr : rrs )
```

Comment:
I wouldn't add all of them. This does not make much sense. I would find the one with matching releases/snapshots flags and also with empty pathMaskPatterns (which is how new ones are created, right?). If no such repo exists, I would fall back to creating a new one. Does it sound right?

Reply:
Yeah, that makes sense, updated.